tests/wpt/tests/html/semantics/embedded-content/media-elements/autoplay-disabled-by-csp.sub.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#eligible-for-autoplay" />
+<title>Test that autoplay is blocked by a document's active sandboxing flags</title>
+</head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/media.js"></script>
+<body>
+<video id="v" autoplay></video>
+<script>
+  async_test((t) => {
+    var v = document.getElementById('v')
+
+    v.addEventListener('playing', t.step_func_done((e) => {
+      e.preventDefault()
+      assert_true(false, 'video should not autoplay due to sandboxing flags');
```

Comment:
This is clearer as `v.addEventListener('playing', t.unreached_func('video should not autoplay due to sandboxing flags'))`